src/nuclio/functions/function-collapsing-row/function-collapsing-row.component.js

```diff
@@ -265,24 +265,38 @@
 
         /**
          * Deletes function from functions list
+         * @param {Object} functionItem
```

Comment:
If this parameter is optional, please change to:
```suggestion
         * @param {Object} [functionItem]
```